### ISSUE TITLE
Schools lookup EdCo keyboard controls

### ISSUE THREAD

**Issue by CarolineRennie** [comicrelief/storybook#102]
When navigating and the edco lookup with screen reader, the enter postcode field is not labelled correctly, the user should be asked to enter postcode or address for their school (which is the copy above the field).

When a user starts to enter their address, the dropdown of matching schools is not announced when it opens so the user would not know they have the option to scroll through the suggested options. 

When the user has pressed enter to select an option from the drop down, there is no audio cue to inform the user that the address has been selected.

When a user enters their address manually, we should indicate which fields are mandatory and which are not - if the * is present we should add the visually hidden description that the field is required.
